public/app/features/serviceaccounts/ServiceAccountPermissions.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Permissions } from 'app/core/components/AccessControl';
+import { contextSrv } from 'app/core/services/context_srv';
+
+import { AccessControlAction, ServiceAccountDTO } from '../../types';
+
+type ServiceAccountPermissionsProps = {
+  serviceAccount: ServiceAccountDTO;
+};
+
+const ServiceAccountPermissions = (props: ServiceAccountPermissionsProps) => {
```

Comment:
It's better to don't mix default and named imports and since we don't have any special reason for default one, it makes sense to use named
```suggestion
export const ServiceAccountPermissions = (props: ServiceAccountPermissionsProps) => {
```